website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -145,7 +145,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.vcs-repo.tags-regex`           | string  | none | A regular expression used to match Git tags. HCP Terraform triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression.|
 | `data.attributes.vcs-repo` | object  | none | Settings for the workspace's VCS repository. If omitted, the workspace is created without a VCS repo. If included, you must specify at least the `oauth-token-id` and `identifier` keys. |
 | `data.attributes.working-directory`             | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository. |
-| `data.attributes.setting-overwrites` | object  | none |   The keys in this object are attributes that have organization-level defaults. Each attribute key stores a boolean value which is `true` by default. To overwrite the default inherited value, set an attribute's value to `false`. For example, to set `execution-mode` as the organization default, set `setting-overwrites.execution-mode` to `false`. |
+| `data.attributes.setting-overwrites` | object  | none |   The keys in this object are attributes that have project-level defaults. Each attribute key stores a boolean value which is `true` by default. To overwrite the default inherited value, set an attribute's value to `false`. For example, to set `execution-mode` as the project default, set `setting-overwrites.execution-mode` to `false`. |
```

Comment:
Should this read to "For example, to set `execution-mode` as the project default, set `setting-overwrites.execution-mode` to `false`." or should it read "For example, to **unset** `execution-mode` as the project default, set `setting-overwrites.execution-mode` to `false`."

The negatives are confusing me right now, especially with my limited knowledge about how this works! 🥴 

cc @emlanctot